data-processing-lib/python/src/data_processing/data_access/data_access.py

```diff
@@ -100,8 +100,9 @@ def get_table(self, path: str) -> tuple[pa.table, int]:
         """
         Get pyArrow table for a given path
         :param path - file path
-        :return: pyArrow table or None, if the table read failed and number of operation retries.
-                 Retries are performed on operation failures and are typically due to the resource overload.
+        :return: Tuple containing
+            pyarrow.Table: PyArrow table if read successfully, None otherwise.
+            the number of retries. Retries are performed on operation failures and are typically due to the resource overload.
         """
         pass
 
```

Comment:
we already have it https://github.com/IBM/data-prep-kit/blob/dev/data-processing-lib/python/src/data_processing/data_access/data_access.py#L99